remoting/error_instance_id.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 Palantir Technologies. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package remoting
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+)
+
+type ErrorInstanceID [16]byte // TODO Can I use existing UUID implementation?
```

Comment:
This is the most widely used one https://github.com/satori/go.uuid

Reply:
@nmiyake and suggestions? I think there is also google uuid package that is popular.

Reply:
Kubernetes uses this one https://github.com/pborman/uuid
This is the google one https://github.com/google/uuid

Reply:
I guess it does not matter that much as long as I am not exposing these types directly.

Reply:
using `google/uuid` for now, it is not exported via the `remoting` package, so it should not be a big deal